snap/README.md

```diff
@@ -161,7 +186,24 @@ $ docker run -it -v"$PWD":/build snapcore/snapcraft:stable bash -c "apt update &
 After building the snap from one of the above methods, you will have a binary snap package called `edgexfoundry_<latest version>_<arch>.snap`, which can be installed locally with the `--devmode` flag:
 
 ```bash
-sudo snap install --devmode edgexfoundry*.snap
+$ sudo snap install --devmode edgexfoundry*.snap
+```
+
+#### Interfaces
+
+After installing the snap, you will need to connect interfaces and restart the snap. The snap needs the `hardware-observe`, `mount-observe`, `system-observe`, and `daemon-notify` interfaces connected. These are automatically connected using snap store assertions when installing from the store, but when developing the snap and installing a revision locally, use the following commands to connect the interfaces:
+
+```bash
+$ sudo snap connect edgexfoundry:hardware-observe
+$ sudo snap connect edgexfoundry:system-observe
+$ sudo snap connect edgexfoundry:mount-observe
+$ sudo snap connect edgexfoundry:daemon-notify
+```
+
+After connecting these restart the services in the snap with:
+
+```bash
+$ sudo snap restart edgexfoundry
 ```
 
-After installing the snap, you will need to connect interfaces and restart the snap as explained in the previous section on [Configuration](https://github.com/edgexfoundry/edgex-go/blob/master/snap/README.md#configuration).
+Note that you cannot install a local snap without using `--devmode` because some of the services require using `systemd-notify` to startup, which in turn requires the `daemon-notify` interface. If this interface is not connected (or the snap is not in devmode), then the services will fail to start and the snap's installation will be aborted. This is not a problem for the snap as downloaded/installed from the store because the assertions to auto-connect the `daemon-notify` interface are included when you download the snap from the store, but no such assertions exist for the locally developed snap because it hasn't been published to the snap store yet.
```

Comment:
Your PR description says that you've switched all the daemons from type=notify to type=forking, so section of text shouldn't be necessary anymore?

Reply:
Yes this is not necessary anymore I will remove it.